app/Controllers/CtrlUser.php

```diff
@@ -146,6 +147,10 @@ public function resendConfirmationEmail(string $userId)
         try {
             $userData = (new UserModel())->getUserDataToResendConfirmEmail($userId);
 
+            if ($userData['confirmed'] !== '0') {
+                throw new UserAlreadyConfirmedException('No es posible enviar el correo de confirmación ya que la cuenta ya ha sido confirmada');
+            }
```

Comment:
We have a constant that represents when a user is not confirmed, replace that magic number for the constant please. 
```suggestion
            if ($userData['confirmed'] !== USER_IS_NOT_CONFIRMED) {
                throw new UserAlreadyConfirmedException('No es posible enviar el correo de confirmación ya que la cuenta ya ha sido confirmada');
            }
```